makejdk.sh

```diff
@@ -197,24 +239,15 @@ buildAndTestOpenJDKViaDocker()
   fi
 
   echo "${info}Using Docker to build the JDK${normal}"
+  
+  createPersistentDockerDataVolume
```

Comment:
minor: createPersistentDockerDataVolume => createPersistentDataVolume
It contains scripts that will be used inside the container so no Docker related data